jbt-it/neomdb
# Server: Restructure TypeOrm

- Moved entities and types
- Removed legacy repository files

How to test:
- Run all tests (check that you can run authTests, memberTests and traineeTests in succession)
- Run server and client and check that login works
